beetsplug/subsonicupdate.py

```diff
@@ -25,16 +25,68 @@
 """
 from __future__ import division, absolute_import, print_function
 
-from beets.plugins import BeetsPlugin
-from beets import config
-import requests
-import string
 import hashlib
 import random
+import string
+
+import requests
+
+from beets import config
+from beets.plugins import BeetsPlugin
 
 __author__ = 'https://github.com/maffo999'
 
 
+def build_payload():
+    """ To avoid sending plaintext passwords, authentication will be
+    performed via username, a token, and a 6 random
+    letters/numbers sequence.
+    The token is the concatenation of your password and the 6 random
+    letters/numbers (the salt) which is hashed with MD5.
```

Comment:
While this is useful context as well, maybe this docstring should first say a little more directly what the function _does_—that is, what the returned dict is supposed to be useful for, and mentioning the fact that the stuff it uses comes from the configuration.